lib/SPIRV/SPIRVRegularizeLLVM.cpp

```diff
@@ -309,14 +309,36 @@ bool SPIRVRegularizeLLVM::regularize() {
           }
         }
         if (auto Cmpxchg = dyn_cast<AtomicCmpXchgInst>(&II)) {
-          Value *Ptr = Cmpxchg->getPointerOperand();
+          // Transform
+          // %1 = cmpxchg i32* %ptr, i32 %comparator, i32 %0 seq_cst acquire
+          // %2 = extractvalue { i32, i1 } %1, 1
```

Comment:
The code below is not doing anything with `extractvalue` anymore, so perhaps you can remove this line?

Reply:
removed